src/telegrafs/actions/thunks.ts

```diff
@@ -149,8 +164,29 @@ export const addTelegrafLabelsAsync = (
   labels: Label[]
 ): AppThunk<Promise<void>> => async (dispatch): Promise<void> => {
   try {
-    await client.telegrafConfigs.addLabels(telegrafID, labels as ILabel[])
-    const telegraf = await client.telegrafConfigs.get(telegrafID)
+    if (labels.length === 0) {
+      throw new Error('No label found to add')
+    }
+
+    const response = await postTelegrafsLabel({
+      telegrafID,
+      data: {labelID: labels[0].id},
```

Comment:
this relates to the previous comment about needing to add all the labels, not just the first one